src/getting-started.md

```diff
@@ -82,13 +82,15 @@ git clone https://github.com/Sjors/bitcoin.git
 cd bitcoin
 git checkout sv2
 ```
-Next, compile the template provider:
+Next, compile the template provider. You can accelerate the process by using multiple cores.
 
 ```bash
 make clean
-./autogen.sh && ./configure 
-make -j 10
+./autogen.sh && ./configure
+nproc # find out how many cores are available on your CPU
+make -j <nproc> # replace the result of nproc here
```

Comment:
Shouldn't we just give an example or a good default?

Reply:
I'll update it to automate this

Reply:
https://github.com/stratum-mining/stratumprotocol.org/pull/204/commits/c406f7a85937f2bf74dfba9d02a38bcd6c0c1a42 lmk what you think @pavlenex

Reply:
Sounds good but I have yet to test it.